content/en/blog/2022/k8sattributes.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Improved troubleshooting using k8s attributes
+linkTitle: Kubernetes attributes
+date: 2022-06-23
+author: Ruben Vargas
+spelling: cSpell:ignore k8sattributes K8sattributes K8sprocessor KUBE
+---
+
+Attaching kubernetes resource attributes to the traces is something that is very
+useful it help us to excctly identify which resource is failing or having
+performance problems. It could be also good to have this information if we want
+to correlate traces with other signals like metrics and logs.
+
+In this article you will learn how to configure the OpenTelemetry collector to
+use the k8sattributes processor, and we will explore how to configure it in
+different scenarios. At the end, we present some alternatives and possible
+future improvements.
```

Comment:
All good, except the part of the processor, what I want to mean is some improvements on how to do this in general.  because there is a way to do it with the resource detector and some operator features.